### ISSUE TITLE
Display problem with crossing 180th meridian

### ISSUE THREAD

**Issue by sikmir** [tumic0/GPXSee#255]
### Steps to reproduce
Let's get some track crossing 180th meridian - [Chukotka 2013](https://www.gpsies.com/map.do?fileId=yilnnniudxemvvaf):
```sh
$ curl 'https://www.gpsies.com/download.do' \
    --data 'fileId=yilnnniudxemvvaf&dataType=1' > chukotka2013.gpx
$ gpxsee chukotka2013.gpx
```
### Actual behaviour
The track is broken by 180th meridian into 2 parts:
![scr1](https://user-images.githubusercontent.com/688044/69787869-57ab4a00-11ce-11ea-8152-babdd91ab13a.jpg)
![scr2](https://user-images.githubusercontent.com/688044/69787870-5843e080-11ce-11ea-86b0-de04291e2a46.jpg)
### Expected behaviour
The whole track should be viewable.
Let's shift whole track by 10° to fit into view:
```sh
$ sed -e '1s/^/cat << EOF\n/;s/\r//;s/lon="\([[:digit:]]\+.[[:digit:]]\+\)"/lon="$(echo \'\1-10\' | bc -l)"/;s/lon="-\([[:digit:]]\+.[[:digit:]]\+\)"/lon="$(echo \'350-\1\' | bc -l)"/;$ a EOF' chukotka2013.gpx | sh > chukotka2013_shifted.gpx
$ gpxsee chukotka2013_shifted.gpx
```
![scr3](https://user-images.githubusercontent.com/688044/69788232-1c5d4b00-11cf-11ea-9994-3d03f2033d56.jpg)

**Comment by tumic0** [comment]
You mean like the map should "wrap" on the edge? Because what I see is a correctly displayed track on both "sides" of the map.

**Comment by sikmir** [comment]
Yes, from UX point of view, there is nothing special within antimeridian, the Earth is round. Most web maps (e.g. [osm.org](https://www.openstreetmap.org/#map=6/66.657/179.692)) allow wrapping, for example I can seamlessly draw track crossing the date line in [nakarte.me](https://nakarte.me/#m=6/67.34832/178.94531&l=O&nktl=HU0-o35sU5-0nu7-Osthng):
![scr1](https://user-images.githubusercontent.com/688044/69851612-28aadc00-1293-11ea-8a3b-ba8bcd789795.jpg)
That's definetly a more natural way, and it would be great if GPXSee could handle it in such a way.

**Comment by tumic0** [comment]
I have no idea how much work this would be (and if it would not require changes outside the map provider) but I'm almost sure it would be enaugh work to outweigh the importance of such a change. How many tracks crossing the Antimeridian are out there?!

**Comment by sikmir** [comment]
> but I'm almost sure it would be enaugh work...

I thought it's just about some additional recalculation, e.g. if E185 tile is required - use W(360-185) instead and so on. But maybe I'm wrong.

> How many tracks crossing the Antimeridian are out there?!

I see 3 real use cases:
* Hiking track in Chukotka
* Trans-Pacific flight/sailing track
* Using GPXSee as general map explorer, rather than gps track viewer
